src/components/RouteMaps/routeData.js

```diff
@@ -0,0 +1,161 @@
+      startName: "SW Corner of the Museum of Nature & Science",
+      startUrl: "https://goo.gl/maps/1qKccJXAj2AkcUDR9",
+      img: cityPark,
+    },
+    {
+      name: "Downtown Denver",
+      length: 8,
+      avgSpeed: 10,
+      rating: "Black",
+      elevation: 180,
+      startName: "Civic Center Park near 14th and Bannock",
+      startUrl: "https://goo.gl/maps/Wvd2p7gUUE3dwSyR7",
+      img: denverDowntown,
+    },
+    {
+      name: "Sloan's Lake",
+      length: 5,
+      avgSpeed: 6,
+      rating: "Green",
+      elevation: 40,
+      startName: "Sloan's Lake Boat House",
+      startUrl: "https://goo.gl/maps/DAKHAQpKsjEmvKYq5",
+      img: sloansLake,
+    },
+    {
+      name: "Central Park",
+      length: 8,
+      avgSpeed: 8,
+      rating: "Blue",
+      elevation: 156,
+      startName: "Pavilion at Central Park",
+      startUrl: "https://goo.gl/maps/54qgNR8kUiHvVQ67A",
+      img: centralPark,
+    },
+    {
+      name: "Cherry Creek",
+      length: 9,
+      avgSpeed: 10,
+      rating: "Black",
+      elevation: 193,
+      startName: "Pavilion at Cheesman Park",
+      startUrl: "https://goo.gl/maps/re1sVufuwRK46s7j6",
+      img: cherryCreek,
+    },
+    {
+      name: "Berkley",
+      length: 6,
+      avgSpeed: 8,
+      rating: "Blue",
+      elevation: 151,
+      startName: "Elitch Carousel",
+      startUrl: "https://goo.gl/maps/TcLGXf1sreCSfTtB8",
+      img: berkley,
+    },
+    {
+      name: "Golden",
+      length: 8,
+      avgSpeed: 8,
+      rating: "Blue",
+      elevation: 194,
+      startName: "Parfet Park",
+      startUrl: "https://goo.gl/maps/7hNycdiaiMdKnYCD8",
+      img: golden,
+    },
+    {
+      name: "Lakewood Gulch",
+      length: 10,
+      avgSpeed: 8,
+      rating: "Blue",
+      elevation: 356,
+      startName: "Denver Skate Park",
+      startUrl: "https://goo.gl/maps/RBH3rkQNswPRLXbD7",
+      img: lakewoodGulch,
+    },
+  ],
+  experimental: [
+    {
+      name: "Arvada Easy Mode",
+      length: 7,
+      avgSpeed: 8,
+      rating: "Blue",
+      elevation: 111,
+      startName: "Arvada Memorial Skate Park",
+      startUrl: "https://goo.gl/maps/L4VmRazTidLH9gWp9",
+      img: arvadaEasy,
+    },
+    {
+      name: "Belmabrook",
+      length: 5,
+      avgSpeed: 8,
+      rating: "Blue",
+      elevation: 153,
+      startName: "Heritage Lakewood Building",
+      startUrl: "https://goo.gl/maps/3cXb5b693HxTJdPk9",
+      img: belmabrook,
+    },
+    {
+      name: "Boulder",
+      length: 9,
+      avgSpeed: 8,
+      rating: "Blue",
+      elevation: 163,
+      startName: "Valmont Skate Park",
+      startUrl: "https://goo.gl/maps/eg35HiPMHLCy9E7o9",
+      img: boulder,
+    },
+    {
+      name: "Crown Hill",
+      length: 7,
+      avgSpeed: 8,
+      rating: "Blue",
+      elevation: 170,
+      startName: "Crown Hill main parking lot",
+      startUrl: "https://goo.gl/maps/tWRe872AJfeRjqX17",
+      img: crownHill,
+    },
+  ],
+};
```

Comment:
This will be our source of truth for route data moving forward. I feel like we may actually be able to remove the `csv` (though thanks for working on it!) just to reduce redundancy. My goal with these refactors is to make it so that we only need to update information in one place instead of across multiple files/code blocks in the repo.